src/test/ChunkPrefetchIterator.test.ts

```diff
@@ -0,0 +1,164 @@
+import { expect } from "chai";
+
+import { TCZYX } from "../loaders/zarr_utils/types";
+import ChunkPrefetchIterator from "../loaders/zarr_utils/ChunkPrefetchIterator";
+
+const EXPECTED_3X3X3X3 = [
```

Comment:
it's hard to understand why this is called 3x3x3 when it has all of t,z,y,x and the arrays are 5 elements each.